### ISSUE TITLE
[Question] Is a manual planned?

### ISSUE THREAD

**Issue by Finnchen123** [ValkyrieofNight/EnvironmentalTech#260]
I'm currently playing on the pack ATM6 (Version 1.4.1) and with the modversion 3.0.6.1 and I wanted to ask if a manual is planned for the different tiers or the modifiers? 

I rly enjoy this mod but it is a bit hard for me to determine which modifier is used for what purpose (I think they all match the ET2 modifiers but there are new mechanics and I personally don't know all modifiers by heart). I know this mod is currently in alpha (I read this in another issue) so I'm rly looking forward to the beta/release/whatever of this mod.

**Comment by ValkyrieofNight** [comment]
Yeah a guide is in development, I've currently mostly been focusing on bug fixes but I will be continuing work on the guide soon.

**Comment by JustJinxed** [comment]
At least a shift click output in the console like version 2 had, would be nice. I have no idea what blocks are needed being new to this mod. And the video for version 3... doesn't say either. Nowhere on the mod page is this info given, nada. 

Very poor
